packages/boxed-expression-component/src/selection/BeeTableSelectionContext.tsx

```diff
@@ -103,6 +103,7 @@ export type BeeTableCellStatus = {
 };
 
 export interface BeeTableCellRef {
+  columnSpan: number;
```

Comment:
I think it is 'rowSpan', isn't?

Reply:
this was removed, the PR was simplified, I am not extending selection API anymore